src/main/java/org/folio/processing/mapping/mapper/writer/common/JsonBasedWriter.java

```diff
@@ -36,9 +37,11 @@
 @SuppressWarnings("rawtypes")
 public class JsonBasedWriter extends AbstractWriter {
   private static final Logger LOGGER = LoggerFactory.getLogger(JsonBasedWriter.class);
-  private ObjectMapper objectMapper = new ObjectMapper();
-  private String entityType;
+  private static final char DOT_SYMBOL = '.';
+  private final ObjectMapper objectMapper = new ObjectMapper();
+  private final String entityType;
   private JsonNode entityNode;
+  private boolean isAlreadyRemovedForExchange;
```

Comment:
It is not a good idea to store the state in a writer. What should be if we try to process several fields with Exchange?
If you need storing state, you can use Value object, cause it incapsulated for a single field and doesn’t affect other fields

Reply:
I think that this case will not affect other fields, because this flag re-writes every time in method`s entry (in 'writeRepeatableValue' method).

But I moved this state to the RepeatableFieldValue, and it looks like more clear for me. 
Thank you!

@OleksiiKuzminov Could you please doublecheck my approach?